### ISSUE TITLE
Unable to deploy with a postgres password that has spaces

### ISSUE THREAD

**Issue by davidleechen** [opf/helm-charts#9]
Abbreviated config fed to helm install/upgrade with -f
```
postgresql:
  bundled: false
  connection:
    host: hostname
    port: 5432
  auth:
    username: postgres
    database: openproject
    password: "This is a password with spaces."
```

Error from the pod
```
-----> Setting PGVERSION=13 PGBIN=/usr/lib/postgresql/13/bin PGCONF_FILE=/etc/postgresql/13/main/postgresql.conf
-----> Starting the all-in-one OpenProject setup at /app/docker/prod/supervisord...
/usr/local/lib/ruby/3.1.0/uri/rfc3986_parser.rb:67:in `split': bad URI(is not URI?): "postgresql://postgres:This is a password with spaces.@postgres-postgresql.production.svc.cluster.local:5432/openproject" (URI::InvalidURIError)
	from /usr/local/lib/ruby/3.1.0/uri/rfc3986_parser.rb:72:in `parse'
	from /usr/local/lib/ruby/3.1.0/uri/common.rb:188:in `parse'
	from /usr/local/lib/ruby/3.1.0/uri/common.rb:692:in `URI'
	from -e:1:in `<main>'
```

I'm pretty sure this password value in the template needs to be piped to `quote`: https://github.com/opf/helm-charts/blob/main/charts/openproject/templates/secrets.yaml#L12

**Comment by oliverguenther** [comment]
Hi @davidleechen , the problem is not just with with quoting the value, but that the value is being templated into a URL. I've not found a templating function to deal with this directly. The alternative is to provide a URL-encoded (percent encoding https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) password.

**Comment by oliverguenther** [comment]
This has been fixed in #46

**Comment by davidleechen** [comment]
Great thanks 👍
